sdk/samples/iot/aziot_realtek_amebaD/aziot_realtek_amebaD.ino

```diff
@@ -0,0 +1,385 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <string.h>
+#include <stdbool.h>
+#include <NTPClient.h>
+#include <sntp/sntp.h>
+
+#include <cstdlib>
+
+#include <WiFi.h>
+#include <PubSubClient.h>
+
+#include <mbedtls/base64.h>
+#include <mbedtls/sha256.h>
+
+#include <az_result.h>
+#include <az_span.h>
+#include <az_iot_hub_client.h>
+
+#include "iot_configs.h"
+#include "ca.h"
+
+// Status LED: will remain high on error and pulled high for a short time for each successful send.
+#define LED_PIN 2
+#define sizeofarray(a) (sizeof(a) / sizeof(a[0]))
+#define MQTT_PACKET_SIZE 1024
+
+static const char* ssid = IOT_CONFIG_WIFI_SSID;
+static const char* password = IOT_CONFIG_WIFI_PASSWORD;
+static const char* host = IOT_CONFIG_IOTHUB_FQDN;
+static const char* device_id = IOT_CONFIG_DEVICE_ID;
+static const az_span device_key = AZ_SPAN_FROM_STR(IOT_CONFIG_DEVICE_KEY);
```

Comment:
any reason why you don't leave this as a `char*` like the rest and create the span locally to the function call?

Reply:
Yes, it's to make it more straightforward to use with the functions provided by Realtek (they implemented them using az_span).